tests/src/Simple/HelloWasm/ILHelpers.il

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.assembly extern mscorlib
+{
+  .publickeytoken = (B7 7A 5C 56 19 34 E0 89 )
+  .ver 4:0:0:0
+}
+
+.assembly extern System.Private.CoreLib
+{
+  .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A )
+  .ver 4:0:0:0
+}
+
+.assembly ILHelpers { }
+
+.class public ILHelpers.ILHelpersTest
+{
+.method public static void EatArgs(string, object)
+{
+ret
+}
+
+.method public static void EatArg(object)
+{
+ret
+}
+
+.method public static int32 InlineAssignByte()
```

Comment:
Both of these test methods are pretty complex. Please add comments on what they're doing.